### PR TITLE
scrot: add page

### PR DESCRIPTION
**Pull request by dakuten**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines

### CODE REVIEW

File: pages/common/scrot.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# scrot
+
+> Capture screenshots using the command line.
+
+- Capture the whole screen:
+
+`scrot`
+
+- Select a window with a mouse then capture the window:
```
Comment (sbrl) [inline pages/common/scrot.md]: `a mouse`? Does that mean there's more than one cursor one the screen? :stuck_out_tongue: :-)

File: pages/common/scrot.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+`scrot -s`
+
+- Wait *sec* seconds before taking a screenshot and display a countdown:
```
Comment (sbrl) [inline pages/common/scrot.md]: Perhaps it would work better giving a concrete example in this case. Say, 5 seconds?

**Comment by dakuten** [comment]
Yes sorry, I didn't see the typo and a quick look on Wikipedia led me think that it was available elsewhere. It's corrected now.

File: pages/linux/scrot.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# scrot
+
+> Capture screenshots using the command line.
```
Comment (agnivade) [inline pages/linux/scrot.md]: "from the command line" might sound better. Let me know what do you think.

File: pages/linux/scrot.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+- Capture the whole screen:
+
+`scrot`
+
+- Select a window with the mouse then capture the window:
```
Comment (agnivade) [inline pages/linux/scrot.md]: Does this mean the capture waits until you click on a window with the mouse ? If yes, we should probably clarify that.

File: pages/linux/scrot.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+
+`scrot -s`
+
+- Wait 5 seconds before taking a screenshot and display a countdown:
+
+`scrot -d 5 -c`
```
Comment (agnivade) [inline pages/linux/scrot.md]: 5 within tokens ({{}})

**Pull request closed without merging**